source/fundamentals/linq.txt

```diff
@@ -320,6 +320,71 @@ following documents:
    { "date" : ISODate("2012-12-05T00:00:00Z"), "grade" : "A", "score" : 13 }
    { "date" : ISODate("2012-05-17T00:00:00Z"), "grade" : "A", "score" : 11 }
 
+Nested Statements
++++++++++++++++++
+
+You can chain or nest ``Select`` and ``SelectMany`` statements. The examples
+in this section use the ``Guidebook`` class, which contain an array of ``Restaurant``
+objects. The following code shows an example of a document modeled by the 
+``Guidebook`` class:
```

Comment:
Q: do you need to use another class? I think it makes this section a little long compared to the others; if you do need Guidebook, you could move it to "Overview"